docs/02-usage.md

```diff
@@ -0,0 +1,14 @@
+# Usage
+
+After the application has pushed and the device has downloaded the latest changes you're ready to go! 
```

Comment:
missing comma
"After the application has pushed and the device has downloaded the latest changes, you're ready to go! "